analyzers/CyberChef/requirements.txt

```diff
@@ -0,0 +1 @@
+dnyspython
```

Comment:
Can you add cortexutils to these requirements?

Reply:
Sure thing -- it's been added.